packages/flutter/lib/src/widgets/view.dart

```diff
@@ -105,17 +283,19 @@ class View extends StatelessWidget {
   /// moved to render into a different [FlutterView] then before). The context
   /// will not be informed when the _properties_ on the [FlutterView] itself
   /// change their values. To access the property values of a [FlutterView] it
-  /// is best practise to use [MediaQuery.maybeOf] instead, which will ensure
+  /// is best practice to use [MediaQuery.maybeOf] instead, which will ensure
   /// that the `context` is informed when the view properties change.
+  /// {@endtemplate}
   ///
   /// See also:
   ///
-  ///  * [View.of], which throws instead of returning null if no [FlutterView]
+  ///  * [RawView.of], which throws instead of returning null if no [FlutterView]
```

Comment:
As discussed offline: Let's remove all the `of` methods from RawView and only have them on the View.

Reply:
Done.